pwa/src/components/welcomeModal/welcomeModal.tsx

```diff
@@ -28,35 +28,51 @@ const WelcomeModal: React.FC = () => {
   const welcomeModalBody = (): JSX.Element => {
     return (
       <>
-        <p>Welcome to the gateway user-interface.</p>
-        <p>It seems you haven't created a application yet. <br />
+        <p>Welcome to Conductor!</p>
+        <p>
+          It seems you haven't created a application yet, in order to consume your API's you will need to create
+          applications.
+        </p>
+        <p>
+          I’ve this is your first time in Conductor we suggest that you follow the
```

Comment:
_I've_ => _If_

Reply:
Maybe:

"Is this your first time using Conductor? [Follow this short tutorial](https://google.nl)."